src/ucp/tag/tag_match.c

```diff
@@ -54,6 +54,15 @@ ucs_status_t ucp_tag_match_init(ucp_tag_match_t *tm)
 
 void ucp_tag_match_cleanup(ucp_tag_match_t *tm)
 {
+    ucp_recv_desc_t *rdesc, *tmp_rdesc;
+
+    ucs_list_for_each_safe(rdesc, tmp_rdesc, &tm->unexpected.all,
+                           tag_list[UCP_RDESC_ALL_LIST]) {
+        ucs_warn("TM receive descriptor %p was not matched", rdesc);
```

Comment:
ucs_diag("unexpected tag-receive descriptor %p was not matched", rdesc);

some MPI apps could have leftovers, UCX should stay silent

Reply:
but UCX is not silent for such errors now. Some errors should be printed during pool cleanup

Reply:
ok
so just need to reword the string

Reply:
done